services/api/README.md

```diff
@@ -11,6 +11,7 @@ The worker can be configured using environment variables. They are grouped by sc
 Set environment variables to configure the application (`API_` prefix):
 
 - `API_HF_AUTH_PATH`: the path of the external authentication service, on the hub (see `HF_ENDPOINT`). The string must contain `%s` which will be replaced with the dataset name. The external authentication service must return 200, 401, 403 or 404. Defaults to "/api/datasets/%s/auth-check".
+- `API_HF_AUTH_BYPASS_PUBLIC_KEY`: the public key used to decode the JWT that can be passed in the 'X-Api-Key' header to bypass the external authentication check. If not set, we don't check if a JWT has been passed in the request. Defaults to empty.
```

Comment:
As @XciD pointed out, it should get the JWT public key directly from asking /api/keys/jwt

Reply:
You can look here: https://github.com/huggingface/spaces-app-manager/blob/main/sse-api/sse_api/config.py#L24-L51

For an example of how to parse the key from moon

Reply:
Done